### PR TITLE
14-create-static-page

### PR DESCRIPTION
**Pull request by Kevin93VuVanManh**
Create static page
Login/Log out/ Sign in
Static page

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,10 +1,11 @@
 source 'https://rubygems.org'
 
-
+gem 'devise'
```
Comment (rikaimind-BuiMinhThu) [inline Gemfile]: Use `"`  instead of `'`  in all files

File: app/assets/javascripts/home.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (rikaimind-BuiMinhThu) [inline app/assets/javascripts/home.coffee]: Remove this file

File: app/assets/stylesheets/home.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the home controller here.
```
Comment (rikaimind-BuiMinhThu) [inline app/assets/stylesheets/home.scss]: Remove this file

File: app/helpers/home_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (rikaimind-BuiMinhThu) [inline app/helpers/home_helper.rb]: Remove this file

File: config/database.yml.example [hunk 5]
```diff
@@ -1,50 +0,0 @@
-development:
```
Comment (rikaimind-BuiMinhThu) [inline config/database.yml.example:1]: don't delete this file!

Remove the `config/.DS_Store` file

File: test/controllers/home_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```
Comment (rikaimind-BuiMinhThu) [inline test/controllers/home_controller_test.rb:1]: In the first project, it is not need to create test unit!
So remove all files in `test` folders then add below line to `.gitignore`

```
test/*
```

**Comment by Kevin93VuVanManh** [comment]
Have fixed

**Pull request closed without merging**